docs/reporting/teamcity.adoc

```diff
@@ -2,22 +2,24 @@
 
 == Installation
 . https://confluence.jetbrains.com/display/TCD9/Installing+Additional+Plugins[Install Allure TeamCity plugin].
```

Comment:
lets also update the link to https://confluence.jetbrains.com/display/TCD10/Installing+Additional+Plugins